khal/controllers.py

```diff
@@ -27,7 +27,10 @@
 from collections import OrderedDict, defaultdict
 from shutil import get_terminal_size
 
-import pytz
+try:
+    import zoneinfo as ZoneInfo
+except ImportError:  # I am not sure if this is correct for the backport
+    from backports import zoneinfo as ZoneInfo
```

Comment:
```suggestion
    from backports.zoneinfo import ZoneInfo
```

Reply:
When flake8 runs on Python3.9, this library won't be installed. It's safe to ignore this specific error.